src/components/reviews/reviewPhoto.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import Modal from "../Modal.jsx";
+
+import "./reviews.css";
+
+class ReviewPhoto extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      modalOpen: false
+    }
+    this.openModalBound = this.openModal.bind(this);
+    this.exitModalBound = this.exitModal.bind(this);
+  }
+
+  openModal() {
+    this.setState({
+      modalOpen: true
+    })
+  }
+
```

Comment:
Im not sure if theres any downside to this but the way i implemented the modal clicks is just this.setState({modalOpen: !this.state.modalOpen}). Might be something to consider in the future.